src/lib.rs

```diff
@@ -26,6 +26,8 @@ pub struct CodegenConfig {
     pub shrink_level: u32,
     /// 0, 1, 2 correspond to -O0, -O1, -O2, etc.
     pub optimization_level: u32,
+    /// If set, the namess section is emitted.
+    pub debug_info: bool,
```

Comment:
It is called `DebugInfo` in Binaryen's C API, though it seems to only control the names section. I think it may make sense to call this `keep_names_section` instead.

Reply:
I think that doesn't matter too much. We can't really try to guess binaryen evolution too much. But If we can do a good guess, then it might make more sense to choose the most probable option.

Reply:
It after #18 the shims are rewritten to avoid using `binaryen-c` then this is controlled more closely by binaryen-rs. Then again, if that happens, one could just make a breaking release of bianryen-rs which changes the name of this variable :)